src/lib/analytics.js

```diff
@@ -157,6 +157,28 @@ window.addEventListener(
   },
 );
 
+// Handle prerenders that have already activated
+if (
+  performance &&
+  performance.getEntriesByType &&
+  performance.getEntriesByType('navigation')[0].activationStart > 0
```

Comment:
I think there's a race condition here. Since the code in this file could run later than the code in the template, it's possible that for prerendered pages a pageview hit will be sent twice.

Consider this case:

1. The page is prerendered but activation happens before the template code has a chance to run
2. When the template code runs the `!document.prerendering` conditional, it returns `true` because the page is no longer prerendering, and a pageview hit is sent.
3. The code in this file runs later and the `activationStart > 0` conditional is also `true`, so another pageview hit is sent.

Reply:
Good spot. Fixed.